src/frontend/Komponenter/Behandling/Totrinnskontroll/FatterVedtak.tsx

```diff
@@ -69,6 +77,24 @@ const FatterVedtak: React.FC<{
             (begrunnelse || '').length > 0 &&
             årsakerUnderkjent.length > 0);
 
+    const hentSimuleringStatus = useCallback(
```

Comment:
```suggestion
    const sammenlignSimuleringsresultater = useCallback(
```

Reply:
er det ikke hent-prefikser det går i for sånne callbacks ?